src/components/preferences/Preferences.js

```diff
@@ -383,8 +384,14 @@ function Preferences(props) {
             </>
         );
     }
-    const validateShortCuts = (values, props) => {
-        const errors = {};
+    const validate = (values, props) => {
+        let errors = {};
+        if (values?.webhook?.url) {
+            const type = values?.webhook?.type?.type;
+            if (type === null || type === undefined || type === "") {
```

Comment:
I think this is a bit simpler:
```suggestion
            if (!values?.webhook?.type?.type) {
```